includes/components/class-attribute.php

```diff
@@ -74,6 +74,9 @@ protected function boot() {
 		add_filter( 'hivepress/v1/meta_boxes', [ $this, 'add_meta_boxes' ], 1 );
 		add_action( 'add_meta_boxes', [ $this, 'remove_meta_boxes' ], 100 );
 
+		// Add archive page redirect.
+		add_action( 'template_redirect', [ $this, 'add_archive_redirect' ], 1000 );
```

Comment:
it's better to name functions like "what it does", not like the task, here "redirect_archive_pages" may be more semantic